src/components/table/examples/table-layout-fitData.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { invoices, Invoice } from './invoices';
+
+/**
+ *
+ * ````tsx
+ * layout="fitData"
```

Comment:
Examples `table-layout-fit-data`  and `table-low-density` have quite similar UI if I'm not mistaken the only difference is a hover effect - for the `table-layout-fit-data` it's gray hover and for the `table-low-density` white hover.  I would suggest renaming the `table-low-density` to  `table-layout-fit-data-interactive-rows` to reduce confusions 



https://user-images.githubusercontent.com/50618208/188556682-badac3a2-473f-4bae-abfc-1f8d27bb3bc6.mov

Reply:
No. This this is probably slightly confusing documentations. In the last example, both the class and the layout are applied.

<img width="393" alt="image" src="https://user-images.githubusercontent.com/35954987/188568433-23e1269a-38f6-4c7e-ba93-9fe9558d7012.png">

Reply:
I think the problem is that not only in this example the rows are interactive. Just the interaction is different. 

I like that you added this `note`
![Screenshot 2022-09-06 at 11 09 47](https://user-images.githubusercontent.com/50618208/188595648-9c3c5045-74d4-4dfd-82d6-1cca23e7c414.png)

Reply:
it's true. that's also bugging me. how non-interactive rows are today highlighted on hover and also selectable (highlighted on click) by default could probably be tweaked a little bit. we can talk about it one day. that also comes out of the box.

highlighting on hover actually has usability benefits, so I don't want to remove it completely. Because it helps the users focus and read on the content of one row in a messy table, and don't mix rows up